src/pages/verify/getting-started/oidc-intro.mdx

```diff
@@ -344,6 +348,95 @@ The token will be issued upon successful user identification:
 }
 ```
 
+## Authenticate with Headless
+
+Headless flow is a custom protocol provided by Criipto that is very similar to <a href="https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0.html" target="_blank">CIBA</a> but with a few key differences.
+
+Where CIBA requires that the user is known in advance so that a request may be sent directly to the authentication device, headless instead returns values to the relying part so that they may trigger necessary UI or interaction elements with the user directly.
+
+Headless is an extension to OpenID Connect. However, unlike in other OpenID Connect flows, there is a direct communication between your application and the OpenID Provider (Criipto Verify), without redirects through the user's browser.
+
+Headless supports use cases not covered by other OpenID Connect flows, such as: 
+
+- Rendering UI elements in your own webpage instead of via a Criipto redirect
+- Performing app2app where a browser is not required
+
+Criipto Verify supports Headless for the Swedish BankID
+
+### Example request
+
+The client application shall make an `HTTP POST` request to the backchannel authentication endpoint to ask for end-user authentication. 
+_The Headless flow is only available for confidential clients, and the example below uses [private key JWTs](/verify/guides/privatekey-jwt/) for client authentication._ 
+
+```text
+HTTP POST https://YOUR_SUBDOMAIN.criipto.id/headless/authorize
+Content-Type: application/x-www-form-urlencoded
+
+scope=openid
+&acr_values=urn:grn:authn:se:bankid
+&client_assertion_type=urn:ietf:params:oauth:client-assertion-type:jwt-bearer
+&client_assertion=YOUR_JWT_ASSERTION
+```
+
+### Example response
+
+Successful response will contain the request id, `headless_req_id`:
+
+```text
+HTTP/1.1 200 OK
+Content-Type: application/json
+{ 
+  "headless_req_id" : "3857f8ff-21b9-48ae-a732-a3bd8128a7ae",
+  "expires_in" : 120
+  "[acr_values]": {... idp specific metadata ...}
```

Comment:
The `idp` term here - does it mean `Criipto Verify`, the underlying native e-ID provider, or a combination of both?

Reply:
e-ID provider, so likely better to use that term instead of IdP since that could mean either.
will update